from .kyc_exceptions import KycNotAcceptedError